router/server.go

```diff
@@ -65,6 +65,10 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 	mux.Handle("/api/prometheus/static_board", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.PrometheusStaticBoardHandler))))
 	mux.Handle("/api/prometheus/boards", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.SaveSelectedPrometheusBoardsHandler))))
 
+	mux.Handle("/api/scanPromotheus", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.ScanPromotheusHandler))))
```

Comment:
Thanks for the PR Vijay!!! Could you move this under line 66 where all APIs related to Prometheus are present?
Also please modify the endpoint to be /api/prometheus/scan

Please do the same for grafana.

Reply:
Thank you for the feedback Naveen. I will do that.